### DISCUSSION TITLE
Clock Division

### DISCUSSION THREAD

**Discussion by Ziad-cyber** [The-OpenROAD-Project/OpenROAD#3023] [Q&A] [answered]
How to define a constraint on a generated clock that's divided by 2 from a master clock source?

**Comment by vijayank88** [comment, marked as answer]
OpenROAD flow accepts all standard SDC formats. Just browse it and implement the same.
OpenSTA doc found [here](https://github.com/The-OpenROAD-Project/OpenSTA/blob/master/doc/OpenSTA.pdf)

**Comment by maliberty** [comment]
In particular see create_generated_clock
